resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,117 @@
+  This API generates the APNS token and returns token through the delegate function `didRegisterForRemoteNotificationsWithDeviceToken`.    
+  For more information, see [Registering Your App with APNs](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns?language=objc)
+  2. In `didRegisterForRemoteNotificationsWithDeviceToken`, set the Push identifier by calling the `setPushIdentifier`.  
+  For more information, see [setPushIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics-mobile-services#set-up-push-messaging).
+ 
+### To verify that push token is successfully synced with ECID service, complete the following steps:
+
+{% hint style="warning" %}
+If SDK privacy status is`optedout`, the push identifier will not be set.
+{% endhint %}
+
+   1. Verify push token sync with the Experience Cloud Identity service.  
+   2. To verify Launch your app with the device connected to a Project Griffon session.   
+   3. In the list of events, verify that you have an event with type _SetPushIdentifier_. In the details panel on the right, verify the value of the push token for this device. The value in pushIdentifier is the same value that is sent to the ECID service. For more information, see screenshot below.
+![alt text](../../.gitbook/assets/push_token_to_identity.png "Verify SetPushIdentifier event.")
+   2. Verify that app’s push token is in the Campaign instance that was mapped with the correct Experience cloud id(mid) by clicking _Adobe Campaign -> Administrator -> Channels -> Mobile App (AEP SDK)_.
+   3. Launch your app and, under the mobile application subscribers, review the list of all Subscribers of app.
+   4. Verify the Experience Cloud ID and Registration Token for the user is displayed.
+   
+   ![alt text](../../.gitbook/assets/campaign_app_subscriber_list.png "App subscriber list, verify mid and push token.")
+   
+   5. If you are using Charles, verify that the push token is successfully synced with the ECID service. Check for the **demdex request** which is marked with the red line in the screenshot below. Also verify the successfull resposne(200) for this network call.  
+ 
+ ![alt text](../../.gitbook/assets/push_identifier.png "ECID network request for push token sync.")
+ 
+ 
+ ## Troubleshooting push notification events tracking  
+
+The following events are related to the tracked push notifications:
+  * Impression (Notification is delivered.)
+  * Click (User clicked the notification.)
+  * Open (App opened in response to user's click on notification.)  
+  For more information about implementing the tracking of push notification events, see [Push Tracking](https://helpx.adobe.com/campaign/kb/push-tracking.html).  
+  
+###  Troubleshooting using Project Griffon  
+
+  You can Verify push token sync with the Experience Cloud Identity service on Project Griffon.  
```

Comment:
Please lower case "Verify" --> "verify" and reword as follows:
"You can verify the push token sync with the ECID service on Project Griffon.